examples/quickstart_ibr.py

```diff
@@ -0,0 +1,87 @@
+import asyncio
```

Comment:
``quickstart_``-prefix is for things from the Quickstart section in the documentation. Call it ``ibr_test.py``, to indicate that this is not a normal example but just runs a sequence of tests.